src/pinns_pde_solve.jl

```diff
@@ -216,8 +216,8 @@ to
 """
 function build_loss_function(eqs,_indvars,_depvars)
     # dictionaries: variable -> unique number
-    indvars = Expr.(_indvars)
-    depvars = [d.name for d in _depvars]
+    depvars = [d.val.name for d in _depvars]
```

Comment:
```suggestion
    depvars = [nameof(value(d)) for d in _depvars]
```

Reply:
There are also a few other things like this below.

Reply:
ok, thanks